http/src/main/resources/leo.conf

```diff
@@ -27,6 +27,9 @@ gke {
   hail-batch-app-config {
```

Comment:
This isn't your change, but is `hail-batch-app-config` in the wrong place here? I think it should be in `azure { … }` not `gke { .. }`. Probably my mistake 😞

Reply:
You might be right, happy to move it!